pkg/skaffold/schema/latest/config.go

```diff
@@ -268,6 +268,9 @@ type KanikoBuild struct {
 	// DockerConfig describes how to mount the local Docker configuration into the
 	// Kaniko pod.
 	DockerConfig *DockerConfig `yaml:"dockerConfig,omitempty"`
+
+	// ResourceRequirements define the resource requirements for the kaniko pod
```

Comment:
Comments should start by the name of the field.

Reply:
done